src/shared/components/ConfirmModal/ConfirmModal.tsx

```diff
@@ -38,9 +45,18 @@ const ConfirmModal: FunctionComponent<ConfirmModalProps> = ({
 	isOpen,
 	confirmCallback = noop,
 	className,
+	remember,
 }) => {
 	const { tHtml } = useTranslation();
 
+	const rememberKey = useMemo(
+		() => (remember ? RememberConfirmationKeys[remember] : ''),
+		[remember]
+	);
+
+	const shouldRemember = rememberKey !== '';
+	const [isRemembered, setRemember] = useLocalStorage(rememberKey, false);
```

Comment:
This is also considered a cookie, so you should add an entry in https://meemoo.atlassian.net/wiki/spaces/AVO2/pages/4310925352/Draaiboek+AVO+Release+1.27+-+BaO to make sure to check cookiebot after release to see if it compains about this setting

Reply:
[Done](https://meemoo.atlassian.net/wiki/spaces/AVO2/pages/4310925352/Draaiboek+AVO+Release+1.27+-+BaO#MTQ-Draaiboek-%E2%9E%A1%EF%B8%8F-QAS)